src/Tribe/Importer/File_Importer_Events.php

```diff
@@ -319,40 +340,52 @@ private function build_event_array( $event_id, array $record ) {
 		}
 
 		/**
-		 * Filter the csv event import event meta.
+		 * Filters the event metadata during CSV import, allowing developers to modify the event data before it's processed.
 		 *
 		 * @since 5.12.4
 		 *
-		 * @param array<string|mixed> $event        An array event meta fields.
+		 * @param array<string|mixed> $event  An array event meta fields.
+		 * @param array               $record An event record from the import.
+		 * @param object              $this   The class instance.
 		 *
 		 * @return array<string|mixed> An array of the autodetect results.
 		 */
 		return apply_filters( 'tec_events_csv_import_event_meta', $event, $record, $this );
 	}
 
 	/**
-	 * Filter allowing user to customize the separator used for organizers
-	 * Defaults to comma ','
+	 * Retrieves the separator used between multiple organizers during event import.
+	 * Defaults to comma ','.
+	 *
 	 * @since 4.6.19
 	 *
-	 * @return mixed
+	 * @return mixed The separator used between organizers. Default is a comma (,).
```

Comment:
```suggestion
	 * @return string The separator used between organizers. Default is a comma (,).
```
This is likely a string.